packages/core/src/queries/connector.test.ts

```diff
@@ -27,13 +27,16 @@ describe('connector queries', () => {
 
   it('findAllConnectors', async () => {
     const rowData = { id: 'foo' };
-    const expectSql = sql`
-      select ${sql.join(Object.values(fields), sql`, `)}
-      from ${table}
+
+    /* eslint-disable sql/no-unsafe-query */
+    const expectSql = `
+      select "id", "enabled", "config", "created_at"
+      from "connectors"
+      order by "enabled" desc, "id" asc
```

Comment:
not following the changes for the test here, and we simply add the order by SQL on top of the old one? As for all the rest updates